### PR TITLE
Add CB-Geo MPM project for GSOC 2024

### PR DESCRIPTION
**Pull request by kks32**

### CODE REVIEW

**Review by henrykironde — APPROVED** [review]

**Review by henrykironde — COMMENTED** [review]
The links to the idea list  don't match. Can you clarify please @kks32

File: 2024/ideas-list.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ page of each organization under the NumFOCUS umbrella at this page.
 - [AiiDA](https://github.com/aiidateam/aiida-core/wiki/GSoC-2024-Projects)
 - [ArviZ](https://github.com/arviz-devs/arviz/wiki/GSoC-2024-projects)
 - [biocommons](https://github.com/orgs/biocommons/projects/8/views/1)
-- [CB-Geo MPM]
+- [CB-Geo MPM](https://github.com/cb-geo/mpm/issues/743)
```
Comment (henrykironde) [inline 2024/ideas-list.md:11]: I would recommend this link here https://github.com/cb-geo/mpm/discussions/744

**Review by henrykironde — COMMENTED** [review]
I think we can just point to https://github.com/cb-geo/mpm/discussions/744

**Comment by kks32** [comment]
@henrykironde Sorry, my bad. Fixed the link

**Pull request merged**